notebook_sandbox/horne_extract/requirements.txt

```diff
@@ -0,0 +1,11 @@
+astropy=5.0
+matplotlib==3.5.0
+numpy==1.21.4
+
+# remember the following:
+# git clone git@github.com:jradavenport/kosmos.git OR git clone https://github.com/jradavenport/kosmos
+# fix ".data" error in trace function
+# use location as argument to sys.path.insert()
+
+# will also need to clone specreduce
+# git clone git@github.com:astropy/specreduce.git OR git clone https://github.com/astropy/specreduce
```

Comment:
is this necessary to keep?  As for specreduce, we could probably either add it to the requirements, or assume they have the clone in order to have this notebook (unless someone just manually pulls the notebook).

Reply:
you can `pip` install directly from github and this works for `specreduce` (and is in fact how test environments are built). however, it doesn't look like that works for `kosmos` alas...

Reply:
The `KOSMOS` portion is necessary because the notebook doesn't work with that code as-is. I could be more clear about how to adjust it. I'll remove the `specreduce` portion.